packages/jest-runtime/src/index.js

```diff
@@ -670,6 +671,7 @@ class Runtime {
         // $FlowFixMe
         (localModule.require: LocalModuleRequire),
       ), // jest object
+      ...extraGlobals.map(globalVariable => global[globalVariable]),
```

Comment:
Should we check that `global[globalVariable]` actually resolves to something, and `throw` if it's `undefined`?

Reply:
I'm not sure, if it `undefined` it won't be found on the global context anyway, which means that his test will throw down the line.

Reply:
We're able to provide a clearer error message, though. Instead of e.g. `TypeError: Cannot read property 'sqrt' of undefined` at some point in the test, we'll get `Jest: You have requested `Maht` as a global variable, but it was not present. Please check your config or your global environment` or something like that

Reply:
Where would be a good place to throw that? after the environment's setup? or just in [here](https://github.com/facebook/jest/pull/7454#discussion-diff-240000812R674)

Reply:
in here is fine